src/components/layout/GlobalStyles.ts

```diff
@@ -105,8 +105,8 @@ const GlobalStyles = createGlobalStyle`
     }
 
     &.danger {
-      border-color: #ff5097;
-      color: #ff5097;
+      border-color: #ff62a2;;
```

Comment:
TODO: create global css vars with default colors, so it can be used on wallet for instance

Reply:
https://github.com/gnosis/dex-react/issues/313